test/models/services.ts

```diff
@@ -82,6 +82,18 @@ describe('services.chooseServiceConfig and services.buildService', function () {
             },
           },
         },
+        {
+          name: 'netcdf-service',
+          type: { name: 'turbo' },
+          collections: [{ id: collectionId }],
+          capabilities: {
+            output_formats: ['application/x-netcdf4'],
+            concatenation: true,
+            subsetting: {
+              temporal: true,
```

Comment:
This specific test would be better with `temporal: false` to reproduce the problem from the original ticket where the service doesn't support spatial or temporal subsetting and the request asks for both.